src/lib/index.js

```diff
@@ -483,6 +483,44 @@ lib.setTranslate = function(element, x, y) {
     return transform;
 };
 
+lib.getScale = function(element) {
+
+    var re = /.*\bscale\((\d*\.?\d*)[^\d]*(\d*\.?\d*)[^\d].*/,
```

Comment:
@mdtusz I was getting worried about all the regexes slowing down our code, but looks like they perform admirably (in comparison with `d3`): 

![image](https://cloud.githubusercontent.com/assets/6675409/15545781/912dc916-226b-11e6-8959-49040395be83.png)

See [code](https://github.com/etpinard/plotly.js-benchmarks/blob/master/suites/set-translate/index.js);